sefaria/helper/descriptions.py

```diff
@@ -12,6 +12,38 @@
 from sefaria.system.exceptions import DuplicateRecordError
 from sefaria.model.abstract import SluggedAbstractMongoRecord
 
+def create_era_link(topic, prev_era_to_delete=None):
+    era_slug_map = {
+        "GN": "geon-person",
+        "RI": "rishon-person",
+        "AH": "achron-person",
+        "CO": "modern-person",
+        "KG": "mishnaic-people",
+        "PT": "mishnaic-people",
+        "T": "mishnaic-people",
+        "A": "talmudic-people",
+    }
+
+    isa_object_aggregate_map = {
+        'tosafot': 'group-of-rishon-people'
+    }
+
+    if prev_era_to_delete:
+        to_topic = isa_object_aggregate_map.get(topic.slug, era_slug_map[prev_era_to_delete])
+        prev_link = IntraTopicLink().load({'toTopic': to_topic, 'fromTopic': topic.slug, 'linkType': 'is-a'})
+        if prev_link:
+            prev_link.delete()
+
+    to_topic = isa_object_aggregate_map.get(topic.slug, era_slug_map[topic.get_property('era')])
+    itl = IntraTopicLink({
+        "toTopic": to_topic,
+        "fromTopic": topic.slug,
+        "linkType": "is-a",
+        "dataSource": "sefaria",
+        "generatedBy": "update_authors_data"
+    })
+    itl.save()
+
 
 
 def update_authors_data():
```

Comment:
This function seems to have the same name as another function imported from topics, consider a different name for readability